src/blocks/overlay/render.php

```diff
@@ -10,7 +10,7 @@
 	<!-- wp:buttons -->
 	<div class="wp-block-buttons">
 	<!-- wp:button {"backgroundColor":"base100","className":"is-style-transparent is-share-button"} -->
-	<div class="wp-block-button is-style-transparent is-share-button"><a class="wp-block-button__link has-base-100-background-color has-background wp-element-button">Share this story</a></div>
+	<div class="wp-block-button is-style-transparent is-share-button"><a href="#" class="wp-block-button__link has-base-100-background-color has-background wp-element-button">Share this story</a></div>
```

Comment:
I had to add an href here, otherwise it was not classed as a focusable object, and broke the focus capture.